ups/meas_extensions_simpleShape.table

```diff
@@ -5,10 +5,11 @@
 # included by low-level LSST packages such as utils or daf_base.
 # Any other package whose interface is used should be listed explicitly
 # rather than assuming it will be included recursively.
-setupRequired(meas_algorithms)
+setupRequired(meas_base)
```

Comment:
@natelust Why did you not include these in the final merge? I listed them all for you.